src/components/CreateNimi/CreateNimi.tsx

```diff
@@ -288,10 +291,10 @@ export function CreateNimi({ ensAddress, ensName, provider }: CreateNimiProps) {
                 })}
 
                 <FormGroup>
-                  {formWidgetList.includes(NimiWidgetType.POAP) && (
-                    <PoapButton>
+                  {getValues('widgets').filter((el) => el.type === NimiWidgetType.POAP).length !== 0 && (
```

Comment:
It might be cleaner to use .some js method here
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
I agree Mr. Violet, I will update this.

Reply:
Done.